CI/before_script.msvc.sh

```diff
@@ -194,7 +194,11 @@ download() {
 }
 
 real_pwd() {
-	pwd | sed "s,/\(.\),\1:,"
+	if type cygpath >/dev/null 2>&1; then
+		cygpath -am "$PWD"
+	else
+		pwd # not git bash, Cygwin or the like
+	fi
```

Comment:
why did we drop the sed magic?

Reply:
Likely because it's magic :p

Reply:
I assume that someone with Windows wrote that for a reason...

@ananace care to take a look?

Reply:
The reason for it was because one of the installers wouldn't handle the path correctly on Windows if given regular slashes. I don't remember if it was the boost packaging or Qt that had the issue though, but it wouldn't extract correctly without it.

Of course all the dependencies have been updated since it was written, so that might've been fixed by now

Reply:
@psi29a We did that thing with an inline PowerShell call when we changed the Boost install directory to fix the path length issue, so I guess we made the `sed` magic redundant when we did that.

Reply:
So long as everything works and we know why it works, then I'm happy. :)